Makefile

```diff
@@ -0,0 +1,50 @@
+ESLINT = node_modules/.bin/eslint --config node_modules/sanctuary-style/eslint-es3.json --env es3
+MOCHA = node_modules/.bin/mocha --reporter dot --ui tdd
+NPM = npm
+TRANSCRIBE = node_modules/.bin/transcribe
+XYZ = node_modules/.bin/xyz --repo git@github.com:sanctuary-js/sanctuary-benchmark.git --script scripts/prepublish
+
+
+.PHONY: all
+all: LICENSE README.md
+
+.PHONY: LICENSE
+LICENSE:
+	cp -- '$@' '$@.orig'
+	sed 's/Copyright (c) .* Sanctuary/Copyright (c) $(shell git log --date=short --pretty=format:%ad | sort -r | head -n 1 | cut -d - -f 1) Sanctuary/' '$@.orig' >'$@'
+	rm -- '$@.orig'
+
+README.md: index.js
+	$(TRANSCRIBE) \
+	  --heading-level 4 \
+	  --url 'https://github.com/sanctuary-js/sanctuary-benchmark/blob/v$(VERSION)/{filename}#L{line}' \
+	  -- $^ \
+	| LC_ALL=C sed 's/<h4 name="\(.*\)#\(.*\)">\(.*\)\1#\2/<h4 name="\1.prototype.\2">\3\1#\2/' >'$@'
+
+
+.PHONY: lint
+lint:
+	$(ESLINT) \
+		--env node \
```

Comment:
Nit: This line appears to be indented by `\t\t` rather than `\t` followed by two spaces.